src/font.c

```diff
@@ -290,13 +299,13 @@ GdipGetFontCollectionFamilyList (GpFontCollection *font_collection, INT num_soug
 	if (font_collection->config)
 		gdip_createPrivateFontSet (font_collection);
 
-	for (i = 0; i < font_collection->fontset->nfont; i++) {
+	for (i = 0; i < num_sought && i < font_collection->fontset->nfont; i++) {
```

Comment:
Here (and below), libgdiplus ignored the `num_sought` parameter. This now matches GDI+ behaviour, obviously.